lib/hosts.nix

```diff
@@ -15,6 +16,8 @@
         ++ base
         ++ extraModules
         ++ [
+          sops-nix.nixosModules.sops
```

Comment:
This is only relevant for Steward and Benefice, so it does not belong here. (e.g. Drawbridge should not depend on anything here)
Note that importing this may affect system config, e.g. it could enable a Systemd service
If we really want to import this by default, we should check what the side-effects of doing so are. (and note that this may change on update)

Reply:
I think that sops-nix is core enough that it's cleaner to import this everywhere. The [definition](https://github.com/Mic92/sops-nix/blob/master/modules/sops/default.nix#L299) has nothing in the `config =` that isn't in the `mkIf (cfg.secrets != {})`, other than a manifest file.

Reply:
ok